### ISSUE TITLE
Unable to deploy on Heroku - 

### ISSUE THREAD

**Issue by mortendybdal** [strapi/strapi#770]
**Node.js version**:
9.8.0

**npm version**:
5.6.0

**Strapi version**:
3.0.0-alpha.11.1

**Operating system**:
Mac OS - High Sierra

**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
After deploying Strapi to Heroku and navigating to the /admin page, users are redirected to /admin/plugins/users-permissions/auth/register and a blank page. The console display following error message:
main.js:1 Uncaught Error: only one instance of babel-polyfill is allowed
    at Object.<anonymous> (main.js:1)
    at Object../node_modules/strapi-helper-plugin/node_modules/babel-polyfill/lib/index.js (main.js:1)
    at t (main.js:1)
    at Object.<anonymous> (main.js:1)
    at Object../admin/src/app.js (main.js:1)
    at t (main.js:1)
    at ../strapi-helper-plugin/node_modules/css-loader/index.js??ref--1-1!../strapi-helper-plugin/node_modules/postcss-loader/lib/index.js??ref--1-2!../strapi-helper-plugin/node_modules/sass-loader/lib/loader.js!./admin/src/components/DownloadInfo/styles.scss (main.js:1)
    at main.js:1

**If the current behavior is a bug, please provide the steps to reproduce the problem**
Assuming you have access to a Heroku account as well as a Compose.io mongo database:
1. > strapi new testproject
2.enter db-name, db-port, db-url, db-name, db-password
3. A Strapi instance is now up-and-running locally and has a connection to the Compose.io mongo database.
4. go to config/enviroments/production/database.json and change:
"uri": "${process.env.DATABASE_URI || '' " => "uri": "${process.env.DATABASE_URI || 'uri_string_to_compose_db' " (You can also set the variable in the heroku interface - but it is faster to reproduce this way)
5. > git init
6. >git commit -m "initial commit"
7. >heroku create testproject
8. >git push heroku master
9. Navigate to testproject.herokuapp.com/admin - !! This is where the error occurse !!

**What is the expected behavior?**
That Strapi shows an interface that lets me register the initial admin user.

**Comment by lauriejim** [comment]
Hi @mortendybdal did you follow steps of this article https://blog.strapi.io/deploying-a-strapi-api-on-heroku/ ?

**Comment by mortendybdal** [comment]
Hi @lauriejim,

Yes I followed the instructions in the article - but I used a database from Compose.io instead of mLab. I have also tried to use mLab to see is that made any difference, but that is not the case.

<img width="812" alt="screen shot 2018-03-12 at 20 54 40" src="https://user-images.githubusercontent.com/3764407/37306320-a601b7c2-2637-11e8-9e71-c5fe1724c3d4.png">

**Comment by mortendybdal** [comment]
Seems like @asharamseervi have experienced kind of the same behaviour (See bug #771  - even though his comment is in a bug threat where it does not belong)

**Comment by asharamseervi** [comment]
Yeah @mortendybdal, Thankyou for listing issue. Exactly I'm also facing same issues.

**Comment by lauriejim** [comment]
I will ask to @soupette to take an eye on your issue.

**Comment by asharamseervi** [comment]
Something is there, which is stopping us to create Content Types, from Dashboard to PRODUCTION ONLY. Please put it on High Priority or Test it once your instance for production, and let us know if everything went well for you, or something we're missing at our instance.

**Comment by lauriejim** [comment]
@asharamseervi It's had to be blocked in production.

**Comment by soupette** [comment]
@mortendybdal the `babel-polyfill` error should be fixed with #1033
